assets/src/parse_bundle_ini.js

```diff
@@ -0,0 +1,59 @@
+var BundleTracker = require('webpack-bundle-tracker');
+var iniParser = require('ini-parser');
+var fs = require("fs");
+var path = require("path");
+var logging = require('./logging');
+
+var parseBundleIni = function(iniFile, bundles, base_dir, externals) {
```

Comment:
What are Javascript's docstring conventions? Can you add docstrings for this function, as well as what the different arguments are fo?